spec/defines/openssl_dhparam_spec.rb

```diff
@@ -11,9 +11,9 @@
     end
 
     it 'fails' do
-      expect {
+      expect do
         is_expected.to contain_file('/etc/ssl/dhparam.pem')
-      }.to raise_error(%r{got ['barz'|String]})
+      end.to raise_error(%r{got ['barz'|String]})
```

Comment:
Not sure why I can't make a suggestion, but this is the native way to test for this:
```ruby
is_expected.to compile.and_raise_error(%r{got ['barz'|String]})
```

This repeats all over this file

Reply:
The current syntax is probably a remnant from old rspec/rspec-puppet syntax

Reply:
Very likely. Technically it kind of works, but if we're making changes now we might as well go for the correct way :)